src/states/indiana/main.py

```diff
@@ -0,0 +1,65 @@
+import random
+import re
+from time import sleep
+from ediblepickle import checkpoint
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from tqdm import tqdm
+from common import indiana_key_namer, work_dir, to_list
+
+BASE_URL = 'https://indianavoters.in.gov/CountyContact/Index'
+
+re_county = re.compile(r'Election Board Contact Information\s*(?P<addr>.*?)\s*'
+                       + r'(?:\n\s*PHONE:\s*(?P<phone>[^\n]*)\s*)?(?:\n\s*FAX:\s*(?P<fax>[^\n]*)\s*)?\n\n',
+                       re.MULTILINE + re.DOTALL)
+
+
+@checkpoint(key=indiana_key_namer, work_dir=work_dir)
```

Comment:
Nit: Since you call them that way, I would reverse the order of the two functions in the codebase.